api/src/main/java/org/openmrs/CohortMembership.java

```diff
@@ -144,4 +144,17 @@ public int compareTo(CohortMembership o) {
 		}
 		return ret;
 	}
+	
+	/**
+	 * Indicates if a given cohortMembership object is equal to this one
+	 * 
```

Comment:
Did you just forget to add the @since annotation?

Reply:
Yes. I just added it